app/app.go

```diff
@@ -153,6 +154,12 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	if err := lock.Verify(); err != nil && !conf.NoVerify {
+		return errors.Wrap(err, "cluster lock signature verification failed. Run with --no-verify to bypass verification at own risk")
+	} else if err != nil && conf.NoVerify {
+		log.Warn(ctx, "Ignoring failed cluster lock signature verification due to --no-verify flag", err)
+	}
+
```

Comment:
```suggestion
	err := lock.Verify(); err != nil {
	if !conf.NoVerify {
		return errors.Wrap(err, "cluster lock signature verification failed. Run with --no-verify to bypass verification at own risk")
	}
	
	 if conf.NoVerify {
		log.Warn(ctx, "Ignoring failed cluster lock signature verification due to --no-verify flag", err)
	}

```

Reply:
Not of a fan of this verbose style

Reply:
how is this "more" verbose? is there a go guideline that i can read upon?

Reply:
More lines and more if statements?